### PR TITLE
[INLONG-2908][Agent] Remove the space before the UUID

### LINKED ISSUE #2908: [Bug][Agent] Delete uuid around space
### What happened

delete the space around the uuid

### What you expected to happen

fetceh uuid

### How to reproduce

nospace

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Title Name: [INLONG-2908][agent] fix uuid blank space

where *XYZ* should be replaced by the actual issue number.

Fixes #2908

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java [hunk 1]
```diff
@@ -344,8 +344,19 @@ public static String fetchLocalIp() {
...
+            if (!StringUtils.isEmpty(result) && StringUtils.containsIgnoreCase(result,"UUID")) {
+                String uuidResult = result.split(":")[1].trim();
+                 uuid = AgentConfiguration.getAgentConf().get(AGENT_LOCAL_UUID, uuidResult);
+                return uuid;
+            }
+        } catch (Exception e) {
+            LOGGER.error("fetch uuid  error");
+            e.printStackTrace();
+        }
+        return uuid;
```
Comment (EMsnap) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:359]: optional or emty string would be better

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java [hunk 2]
```diff
@@ -344,8 +344,19 @@ public static String fetchLocalIp() {
...
      */
     public static String  fetchLocalUuid() {
-        String result = ExcuteLinux.exeCmd("dmidecode | grep UUID");
-        return AgentConfiguration.getAgentConf().get(AGENT_LOCAL_UUID, result);
+        String uuid = null;
+        try {
+            String result = ExcuteLinux.exeCmd("dmidecode | grep UUID");
+            if (!StringUtils.isEmpty(result) && StringUtils.containsIgnoreCase(result,"UUID")) {
+                String uuidResult = result.split(":")[1].trim();
+                 uuid = AgentConfiguration.getAgentConf().get(AGENT_LOCAL_UUID, uuidResult);
```
Comment (EMsnap) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:352]: extra blank space ?

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 1 above)
Comment (EMsnap) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:359]: If agent conf has configured uuid,  your method still return null

**Review by EMsnap — COMMENTED** [review]
PLAL

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java [hunk 3]
```diff
@@ -344,8 +344,19 @@ public static String fetchLocalIp() {
...
+                return uuid;
+            }
+        } catch (Exception e) {
+            LOGGER.error("fetch uuid  error");
+            e.printStackTrace();
```
Comment (healchow) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:357]: Please print exception stack trace with LOGGER.error, for example:
`LOGGER.error("fetch uuid error", e);`

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 1 above)
Comment (LvJiancheng) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:359]: ok

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 2 above)
Comment (LvJiancheng) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:352]: ok

Comment (LvJiancheng) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:352]: the result like :      UUID.  :   XXXX-XXX-XXXX
drop the blank space

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 1 above)
Comment (LvJiancheng) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:359]: done

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 3 above)
Comment (LvJiancheng) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:357]: done

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 1 above)
Comment (LvJiancheng) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:359]: Unless the result is null ,the uuid still return null; but whatever the agent conf has configured uuid ,the  "ExcuteLinux" execute all time ,still optimize

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 3 above)
Comment (healchow) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:362]: Please delete `e.printStackTrace();`.

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java [hunk 4]
```diff
@@ -344,8 +344,24 @@ public static String fetchLocalIp() {
      * check agent uuid from manager
      */
     public static String  fetchLocalUuid() {
-        String result = ExcuteLinux.exeCmd("dmidecode | grep UUID");
-        return AgentConfiguration.getAgentConf().get(AGENT_LOCAL_UUID, result);
+        String uuid = "";
+        try {
+            String localUuid = AgentConfiguration.getAgentConf().get(AGENT_LOCAL_UUID);
+            if (!StringUtils.isEmpty(localUuid)) {
```
Comment (healchow) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:350]: Please use `StringUtils.isNotEmtpy()`.

File: inlong-agent/conf/agent.properties [hunk 5]
```diff
@@ -51,6 +51,8 @@ agent.fetcher.classname=org.apache.inlong.agent.plugin.fetcher.ManagerFetcher
 # max wait time(s) for thread pool to terminate running
 thread.pool.await.time=30
 agent.local.ip=127.0.0.1
+agent.local.uuid=
+
```
Comment (healchow) [inline inlong-agent/conf/agent.properties:55]: Remove the blank line.

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java [hunk 6]
```diff
@@ -344,8 +344,24 @@ public static String fetchLocalIp() {
...
+                uuid = localUuid;
+                return uuid;
+            }
+            String result = ExcuteLinux.exeCmd("dmidecode | grep UUID");
+            if (!StringUtils.isEmpty(result)
+                    && StringUtils.containsIgnoreCase(result,"UUID")) {
+                    uuid = result.split(":")[1].trim();
+                    return uuid;
+                }
```
Comment (healchow) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:359]: The indent style is not correct.

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java (hunk 4 above)
Comment (LvJiancheng) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/AgentUtils.java:350]: done

**Review by EMsnap — APPROVED** [review]
LGTM

**Review by healchow — APPROVED** [review]

**Pull request merged**